lib/app/routes/exercises.rb

```diff
@@ -40,7 +40,8 @@ class Exercises < Core
           submission: submission,
           next_submission: next_submission,
           sharing: Sharing.new,
-          solution: src_obj.solution
+          solution: src_obj.solution,
+          analysis: submission.analysis
```

Comment:
don't think you need to pass this analysis, since you can get it from submission.